include/fizzy/fizzy.h

```diff
@@ -145,6 +146,8 @@ typedef struct FizzyImportedFunction
 } FizzyImportedFunction;
 
 /// Validate binary module.
+///
+/// @return true if module is valid, false otherwise.
```

Comment:
Well, even if it is basic, shouldn't the params also be included?

Reply:
Added.